lib/chef/knife/bootstrap/templates/windows-chef-client-msi.erb

```diff
@@ -70,10 +70,6 @@ goto architecture_select
 @set MACHINE_OS=2008
```

Comment:
We don't produce 2008 MSIs either so we may be able to remove this as well. We'd have to see when we ended that support though.